mail/postfix/src/opnsense/service/templates/OPNsense/Postfix/master.cf

```diff
@@ -10,7 +10,11 @@
 # service type  private unpriv  chroot  wakeup  maxproc command + args
 #               (yes)   (yes)   (no)    (never) (100)
 # ==========================================================================
+{%   if helpers.exists('OPNsense.postfix.general.inet_port') and OPNsense.postfix.general.inet_port == '25' %}
```

Comment:
this is why we have the model migration + required Y so the if-else code is not required